array-methods.js

```diff
@@ -67,6 +103,26 @@ var sumOfBankBalances = null;
  */
 var sumOfInterests = null;
 
+function findStates(element) {
+  if (element.state === 'WI' || element.state === 'IL' || element.state === 'OH' || element.state === 'GA' || element.state === 'WY' || element.state === 'DE') {
```

Comment:
Try and see if you can make this shorter by using an array and some of the methods that come with arrays.